src/gitgraph.js

```diff
@@ -55,7 +56,8 @@
       this.template.commit.spacingY = 0;
       this.template.branch.spacingX = 0;
       this.orientation = "horizontal";
-      this.template.branch.labelRotation = -90;
+      this.template.branch.labelRotation =  !(_isNullOrUndefined(options, "template.branch.labelRotation")) ?
```

Comment:
Same here, I prefer to have a "positive" condition than keeping "default value at the end".